dev-environment.yml

```diff
@@ -4,17 +4,17 @@ channels:
   - defaults
 dependencies:
   - python >=3.5
-  - setuptools
-  - pip
+  - setuptools >=20.7
+  - pip >=8.1
   - pytest
   - pytest-cov
   - coverage
   - sphinx
-  - numpy
-  - scipy
+  - numpy >=1.11
+  - scipy >=0.17
   - matplotlib >=2.1
```

Comment:
I think the only reason we have this is because the `Rectangle._angle` changing to `Rectangle.angle`. I think if we use `Rectangle._angle` in the notebook it is forward compatible.